tests/test_topics.py

```diff
@@ -738,6 +738,96 @@ def foo():
 
         asyncio.get_event_loop().run_until_complete(doit())
 
+    def test_controller_command_success(self):
+        """Test ack when a controller command succeeds.
+        """
+        async def doit():
+            async with Harness(initial_state=salobj.State.ENABLED) as harness:
+
+                ackcmd = await harness.remote.cmd_wait.set_start(duration=0, timeout=STD_TIMEOUT)
+                self.assertEqual(ackcmd.ack, salobj.SalRetCode.CMD_COMPLETE)
+
+        asyncio.get_event_loop().run_until_complete(doit())
+
+    def test_controller_command_return_ackcmd(self):
+        """Test ack when controller command callback returns an ackcmd.
+        """
```

Comment:
"returns an ackcmd" or "raises an ackcmd"? The test here seems to be that the `return_ack` raises.

Reply:
Good point.  I'll change these doc strings to "Test exception raised by remote command when controller command callback returns..."